server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/agents/agent_row_widget.js.msx

```diff
@@ -34,15 +34,44 @@ define(['mithril', 'lodash'], function (m, _) {
     return "/go/agents/" + uuid + "/job_run_history";
   };
 
+  var createBuildDetailsDropDown = function (agent, args) {
+    return (
+      <BuildDetailsWidget agent={agent}
+                          dropDownReset={args.dropDownReset}/>
+    );
+  };
+
   var AgentRowWidget = {
+    controller: function (args) {
+      var ctrl = {
```

Comment:
Across all the controllers, we follow the same pattern of returning a ctrl instead of assigning a property. To maintain consistency I did the same here. When I looked at the mithril documentation also most of the examples returns an object instead of assigning a property.